QUARKUS-711.md

```diff
@@ -0,0 +1,20 @@
+# QUARKUS-711 - RestEasy support for Multipart data
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-711
+
+Test plan focuses on RestEasy support for multipart data provided by `resteasy` and `resteasy-multipart` extensions, as described in:
+- https://quarkus.io/guides/rest-json#multipart-support
+- https://quarkus.io/guides/rest-client-multipart
+
+Apart from topics described below, the functionality remains unchanged.
+
+## Scope of the testing
+- Testing verifies:
+    - Correct multipart request data processing.
+    - The new default value of charset of parts of a multipart request is `UTF-8` (in contrast to previous `us-ascii`) if the encoding is not specified
+    - The value of the charset is overridden by property `quarkus.resteasy.multipart.input-part.default-charset`.
+    - The value of content type is specified by property `quarkus.resteasy.multipart.input-part.default-content-type`.
+- Multipart support extracted into a new `resteasy-multipart` extension - all affected applications must migrate by adding this extension.
+
+## Automated test development
+- Add integration tests in Quarkus - verify scenarios not covered by existing unit tests of the `resteasy-multipart` extension.
```

Comment:
There is also `rest-client-multipart-quickstart` QS, was it also checked?

Reply:
https://github.com/quarkusio/quarkus-quickstarts/blob/development/rest-client-multipart-quickstart/pom.xml#L46 ... this dependency should be switched to quarkus one.

You can add QS review as one of the actions you are gonna perform.

Reply:
I would enhance `001-quarkus-getting-started` with multipart scenario TestCase instead of upstream testsuite.
Upstream is covered by Quickstart and RESTEasy TS, adding test in Quarkus upstream testsuite doesn't help us with RHBQ 1.11 as we would need to ask for back-port to 1.11 branch and the doors are closing soon.

Reply:
I didn't check the QS, thanks for pointing it out.

Test case in BeefyTS makes more sense.